### ISSUE TITLE
[BUG] - Upper case problems in title attributes in XML RSS feed

### ISSUE THREAD

**Issue by racalle** [mattstratton/castanet#337]
Hi,

I have detected that lower case are not respected in one of my episodes when copying title from markdown to XML. 

The title in MD is: 

`title = "Desinformación en tiempos de pandemia"`

The resulting attributes in XML are converted to 

`<title>Desinformación en Tiempos De Pandemia</title>` or
`<itunes:title>Desinformación en Tiempos De Pandemia</itunes:title>`

Some specs:

go version is 1.15, and I am using 1.14.1 release as a hugo module. With prior version same behaviour happened.

In order to check online, the feed url is [this one](https://callejongallosciegos.rcalle.es/episode/index.xml), the site is [https://callejongallosciegos.rcalle.es/](https://callejongallosciegos.rcalle.es/)

Copy of MD:

```


+++
Description = "Hablamos de los contenidos negacionistas encontrados en varias RRSS llenos de falsos expertos, falsos analistas, materiales contrarios a las vacunas, anti PCRs, anti mascarillas, etcétera."
date = 2021-01-02T00:00:00+01:00

aliases=["/2"]
episode = "2"

podcast_duration = "01:34:51"
podcast_bytes = "72772994" # the length of the episode in bytes


podcast_file = "002_desinformacion_en_tiempos_de_pandemia.m4a" 

episode_image = "img/episode/002.png"
#episode_banner = "img/episode/default-banner.jpg"

title = "Desinformación en tiempos de pandemia"

#subtitle = ""

images = ["img/episode/default-social.jpg"]

explicit="false"

author = "Ramón Calle"

guests= ["fgallo"] #based on the filename without extension.

#truncate = "" #characters to truncate the summary on the row layout

hosts = ["rcalle"] # The names of your hosts, based on the filename without extension.

#youtube = ""

#media_override # if you want to use a specific URL for the audio file

#PublishDate = 2020-12-12T00:00:00+01:00 # this is the datetime for the when the epsiode was published. This will default to Date if it is not set.

#sponsors = []

+++

---

# Episodio 2. Desinformación en tiempos de pandemia


#### NOTA: las imágenes aquí presentadas son ejemplos de contenidos que consideramos desinformativos. Este podcast es contrario a este tipo de materiales y se opone a la difusión de los mismos.

Les recomendamos que busquen siempre medios que contrasten la información que publican, como por ej. el blog [Ablogios](https://www.ablogios.com) y podcasts como Qarentena o Bacteriófagos, publicados por Carmela García, científica, divulgadora y comunicadora.


----

Para escuchar el audio negacionista del virus de la Covid-19 que sirve de introducción a este episodio vayan al minuto 13.41 del [episodio piloto.](https://callejongallosciegos.rcalle.es/001-piloto/)

En este episodio hablamos de los contenidos negacionistas encontrados en varias RRSS llenos de falsos expertos, falsos analistas, contenidos contrarios a las vacunas, anti PCRs, anti mascarillas.

Ejemplos de imágenes encontradas mientras grabábamos el episodio:

(some images)

```
This translates in wrong case sensitive in different places, example in [itunes](https://podcasts.apple.com/es/podcast/el-callej%C3%B3n-de-los-gallos-ciegos/id1544675741)

Many thanks.

**Comment by mattstratton** [comment]
This is very odd! I don't see anything in the template for the feed that would manipulate the capitalization, but I'll check it out!

**Comment by mattstratton** [comment]
Ah, I was wrong, we use the `title` function built into hugo for this

https://gohugo.io/functions/title/#readout

I see that we do that on the individual episode pages too, but not on the row and grid layouts.

I need to figure out the best way to solve for this; for my own podcast I know I wanted the titles converted to title case, so if I remove this, it will break older implementations.

As I see it, maybe the way to do it is with two things:

1) Update the grid and row layouts to be in sync with episode pages and the feed
2) Add an optional config parameter to override this (like `disable_title_capitalization`) and if this is set, it will not update the capitalizations, but if it's not set, it just works as it did already.

**Comment by racalle** [comment]
@mattstratton I see this solution as a good way to solve this capitalization problem and not modify the look of title when upgrading. 

Maybe "enable_title_capitalization" default to false would be more elegant, but it will break the behavior of older versions like you said.

**Comment by racalle** [comment]
Hello Matt, ¿what type of help can we provide? 

As I can see you have the problem identified. Removing the "title()" calls when a config parameter "disable_feed_title_capitalization" is set to true will repair this bug (or similar approach).

Do you have paypal available? My site is working fine with castanet and I am grateful for it. Thanks!

**Comment by mattstratton** [comment]
Fixed in #370

**Comment by racalle** [comment]
Hello @mattstratton . Finally I published something and I could test. I don't know why but it is not working for the part of RSS XML which is the important for me. 

I added  the parameter in it's section of "config.toml" without sucess :-( 

```
[params]

 #2022 evitar todos los replaces de los cases mal puestos
disable_title_caps="true"
```

¿Any idea why it is not working? Thx for your time.
